app/scripts/lib/setupSentry.js

```diff
@@ -30,6 +30,53 @@ export const SENTRY_BACKGROUND_STATE = {
   AccountTracker: {
     currentBlockGasLimit: true,
   },
+  PermissionLogController: {
+    // double-check if contains PID?
+    permissionHistory: true,
+    // double-check if contains PID?
+    permissionActivityLog: true,
+  },
+  AnnouncementController: {
+    announcements: true,
+  },
+  GasFeeController: {
+    gasFeeEstimates: true,
+    estimatedGasFeeTimeBounds: true,
+    gasEstimateType: true,
+  },
+  TokenListController: {
+    // double-check if contains PID?
+    tokenList: true,
+    tokensChainsCache: true,
+    preventPollingOnNetworkRestart: true,
+  },
+  TokensController: {
+    // double-check if contains PID?
+    tokens: true,
```

Comment:
I'd advice omitting all of these as well, just keeping `{}` for the TokensController so that we can see the property types in the mask

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/b3330828235d61e68ab356e3e859a30d3c44af6f